packages/plus/src/components/CustomerSatisfaction/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { css, keyframes } from '@emotion/react'
+import styled from '@emotion/styled'
+import { Stack } from '@ultraviolet/ui'
+import { useMemo, useState } from 'react'
+import rating1 from './assets/1-5.svg'
+import rating1NS from './assets/1-5NB.svg'
+import rating2 from './assets/2-5.svg'
+import rating2NS from './assets/2-5NB.svg'
+import rating3 from './assets/3-5.svg'
+import rating3NS from './assets/3-5NB.svg'
+import rating4 from './assets/4-5.svg'
+import rating4NS from './assets/4-5NB.svg'
+import rating5 from './assets/5-5.svg'
+import rating5NS from './assets/5-5NB.svg'
+
+const ratings = [
+  {
+    value: 1,
+    imgSelected: rating1,
+    imgNotSelected: rating1NS,
+  },
+  {
+    value: 2,
+    imgSelected: rating2,
+    imgNotSelected: rating2NS,
+  },
+  {
+    value: 3,
+    imgSelected: rating3,
+    imgNotSelected: rating3NS,
+  },
+  {
+    value: 4,
+    imgSelected: rating4,
+    imgNotSelected: rating4NS,
+  },
+  {
+    value: 5,
+    imgSelected: rating5,
+    imgNotSelected: rating5NS,
+  },
+] as const
+
+type Value = (typeof ratings)[number]['value']
```

Comment:
use number because in DX we will always need to cast number to Value type. We should handle number <0 and >5 and transform them to min and max ?

Reply:
@DorianMaliszewski rating is hard coded number of values, if I type it as number it means developer will be able to put a value higher or lower than the number of rating. What is the purpose of doing so? Do you have an example?